datadog_checks_base/datadog_checks/base/utils/tailfile.py

```diff
@@ -116,3 +116,8 @@ def tail(self, line_by_line=True, move_end=True):
             # log but survive
             self._log.exception(e)
             raise StopIteration(e)
+
+    def close(self):
+        if self._f:
+            self._f.close()
```

Comment:
At the moment, if `self._f.close()` the check will fail too, shall we protect against that ?

Reply:
That's already the case here: https://github.com/Datadog/integrations-core/blob/44859cdda277c67cbdf8bdb7ff9b48643cc4b088/datadog_checks_base/datadog_checks/base/utils/tailfile.py#L29 so the behavior is not going to change and introduce new exceptions.